demos.js

```diff
@@ -95,7 +95,7 @@ const allDemos = [
         "group": "Hoodlum and Tristar & Red Sector Inc. and Paradox",
         "youtube": "tTN81zaiP3U",
         "date": "2004-11-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00014/00014107.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/31/2c/66b6.154362.png",
```

Comment:
I'd suggest removing this entry entirely. It's unlikely to be interesting to someone looking to discover 64k intros.